build.rs

```diff
@@ -62,10 +64,13 @@ fn main() -> std::io::Result<()> {
     #[cfg(feature = "serde")]
     {
         config.type_attribute(".", "#[serde(rename_all = \"camelCase\")]");
-        config.type_attribute(".", "#[allow(clippy::doc_lazy_continuation)]");
-        config.type_attribute(".", "#[allow(clippy::empty_docs)]");
     }
 
+    config.type_attribute(".", "#[allow(clippy::doc_lazy_continuation)]");
+    config.type_attribute(".", "#[allow(clippy::empty_docs)]");
```

Comment:
These two are moved out of the `serde` block because I think they were placed behind that feature gate by mistake. A second pair of eyes to confirm this understanding would be helpful.

Reply:
I ended up putting these back in [ccc47d5](https://github.com/meshtastic/rust/pull/52/commits/ccc47d5e0ef5674c0cf735d248114cff8c41b848). I still think they may be incorrectly hidden behind the `serde` feature, but am leaving that unchanged in this PR unless someone can confirm.